packages/react/src/index.tsx

```diff
@@ -39,7 +48,7 @@ export class ConfidenceReact implements EventSender, Trackable, FlagResolver {
 
   /** @internal */
   get contextState(): string {
-    return this.delegate.flagState + Value.serialize(this.delegate.getContext());
```

Comment:
Was this the only use of `flagState`? Maybe we can remove it from the API in that case?